crates/rattler_installs_packages/src/artifacts/sdist.rs

```diff
@@ -43,6 +44,25 @@ pub enum SDistError {
     WheelCoreMetaDataError(#[from] WheelCoreMetaDataError),
 }
 
+/// Pop until the file name is found an use the rest of the path
+fn strip_until_containing_prefix(name: &str, path: &Path) -> PathBuf {
+    // Use up until the last occurrence of the distribution name
+    let components = path
+        .components()
+        .rev()
+        .take_while(|c| {
+            !c.as_os_str()
+                .to_str()
+                // Keep taking if we cannot convert
+                .unwrap_or("")
+                .contains(name)
```

Comment:
`.starts_with` is not correct?

Reply:
Or `==`?

Reply:
`starts_with` would work I suppose, `==` not because `flask` should match for `flask-1.0.0` and such.